### ISSUE TITLE
Podcasts Not Updated

### ISSUE THREAD

**Issue by Anastik** [DotPodcast/dotpodcast-player#107]
When you subscribe to a Podcast the most current episode is from February 23rd. Additionally, under "Subscriptions" you can only view the most current podcast you subscribed to.

**Comment by freethejazz** [comment]
Hey @Anastik, thanks for reporting these issues. We're looking into the problem with the most current episode and think we know the cause of this issue.

Under subscriptions, we have been able to subscribe to multiple podcasts and see them all, can you add some more detail about what you're not seeing. A step-by-step of what you did including the names of the podcasts you're subscribing to would be very helpful.

Thanks and let us know if you need any clarification :)

**Comment by Anastik** [comment]
Yeah, I see that you were able to get the most updated podcasts on there. Nice!

So when I go to "Your Subscriptions" it only shows the most current podcast that I've subscribed to. For instance, I've subscribed to both "Up First" and "The Daily" but it only shows the one I subscribed to most recently. I'm using Google Chrome on an Apple laptop with the most current version of Blockstack running.

And then each time I go back into a podcast, after having subscribed to another, I'm no longer subscribed to the original one. Hopefully that makes sense.
<img width="1275" alt="screen shot 2018-03-12 at 5 47 47 pm" src="https://user-images.githubusercontent.com/33998307/37318140-aa41a406-261d-11e8-9b44-324e8ca2761e.png">

**Comment by freethejazz** [comment]
Hey @Anastik, I wasn't able to replicate this issue locally, but I did make a small change on Friday that could possibly help. Can you try again and see if that resolved the issue for you?

**Comment by Anastik** [comment]
Yeah, unfortunately that didn't fix the issue. And I don't see any new podcasts after March 12th updated in my feed now.
